mautic/mautic
# change list icon on pop up selection when creating an email

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed issues  |  no

## Description
Change list icon to pie chart when creating a segment email
## Steps to test this PR
- get the PR and see the list icon in the popup to select segment email change to pie chart
